FastExcel/DefinedName.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+namespace FastExcel
+{
+    partial class FastExcel
```

Comment:
could this file be renamed to FastExcel.DefineName.cs it helps to identify that it's a partial class